src/pages/ExplorePage/ExplorePage.jsx

```diff
@@ -0,0 +1,54 @@
+import "./ExplorePage.css";
+import { useEffect } from "react";
+import {
+  useFetchingData,
+  useLikeActions,
+  useVideoHistory,
+} from "../../utilities/CustomHooks";
+import { useDataValues } from "../../contextAndReducers/DataProvider";
+import { FilterArea } from "../../components/FilterArea/FilterArea";
+import { VideoCard } from "../../components/VideoCard/VideoCard";
+import { Loader } from "../../components/Loader/Loader";
+import { Link } from "react-router-dom";
+function ExplorePage() {
+  const { fetchData } = useFetchingData();
+  const { state, dispatch } = useDataValues();
+  const { getLikedVideos } = useLikeActions();
+  const { getHistory } = useVideoHistory();
+  useEffect(() => {
+    fetchData();
+    getLikedVideos();
+    getHistory();
+    console.log("run");
+  }, []);
+  return (
+    <div>
+      <FilterArea />
+      {state.loader && (
+        <div className="flex-center height-100">
+          <Loader />
+        </div>
+      )}
+      {!state.loader && (
+        <div className="Explore-main">
+          {state.data.map((item) => (
+            // <Link to="/singleVideo-page" >
+            <VideoCard
+              key={item._id}
+              contentPhoto={item.thumbnail.url}
+              contentPhotoAlt={item.thumbnail.altText}
+              creatorPhoto={item.creatorLogo.url}
+              creatorPhotoAlt={item.creatorLogo.altText}
+              thumbNail={item.title}
+              creatorName={item.creator}
+              id={item._id}
+              obj={item}
```

Comment:
a short way to effectively spread the contents of item is to simply do this
``` <VideoCard  {...item} /> ```
and then you can just go to VideoCard and restructure as 
``` export const VideoCard=({thumbnail,creatorLogo,title,creator,_id})=>{}```

Reply:
I don't wanted to do this since I also wanted the whole object so just kept it simple that way . And I also feel its easy to understand this way that's kept it this way

Reply:
This way you'll get the whole object too.
``` export const VideoCard=(video)=>{
    const {_id,title etc} = video;
}```

Reply:
Ok thanks for the suggestion will keep in mind